internal/daemon/vendor-specific-plugins/marvell/ovs-dp/ovsdp.go

```diff
@@ -0,0 +1,107 @@
+package ovsdp
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/go-logr/logr"
+	mrvlutils "github.com/openshift/dpu-operator/internal/daemon/vendor-specific-plugins/marvell/mrvl-utils"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	ovsDBPath  = "/var/run/openvswitch/db.sock"
+	ovsCLIPath = "/usr/local/bin/ovs-vsctl"
+	deviceID   = "a063"
```

Comment:
I think constants should be CamelCase. Also because there is already `ovsDBPath` (in this spelling) used for something else.